docs/tutorials/multilingual/prepare-iwslt17-multilingual.sh

```diff
@@ -0,0 +1,116 @@
+#! /bin/bash
+
+# Code taken from (modified):
+# https://github.com/pytorch/fairseq/blob/master/examples/translation/prepare-iwslt17-multilingual.sh
```

Comment:
Could you add a small snippet to indicate the original license the file was under + copyright:
```suggestion
# https://github.com/pytorch/fairseq/blob/master/examples/translation/prepare-iwslt17-multilingual.sh
# MIT licensed. "Copyright (c) Facebook, Inc. and its affiliates. All rights reserved."
```

Otherwise no concerns with copying such a file over and/or modifying it.

Reply:
Thanks! Fixed